src/cpu/platform.cpp

```diff
@@ -36,6 +36,7 @@
 #include "cpu/aarch64/cpu_isa_traits.hpp"
 #if DNNL_AARCH64_USE_ACL
 // For setting the number of threads for ACL
+#include "arm_compute/core/CPP/CPPTypes.h"
```

Comment:
Could you update the comment on line 38?

Reply:
okk